db/rdbms/src/test/java/io/camunda/db/rdbms/sql/typehandler/WildcardTransformingStringTypeHandlerTest.java

```diff
@@ -32,4 +32,21 @@ private static Stream<Arguments> provideElasticsearchQueries() {
         Arguments.of("some*query?with*wildcards", "some%query_with%wildcards"),
         Arguments.of("some\\*query\\?with*wildcards?", "some\\*query\\?with%wildcards_"));
   }
+
+  private static Stream<Arguments> provideSqlQueries() {
+    return Stream.of(
+        Arguments.of("some%query", "some\\%query"),
+        Arguments.of("some\\%query", "some\\%query"),
+        Arguments.of("some_query", "some\\_query"),
+        Arguments.of("some\\_query", "some\\_query"),
+        Arguments.of("some%query_with%wildcards", "some\\%query\\_with\\%wildcards"),
+        Arguments.of("some\\*query\\_with%wild*cards?", "some\\*query\\_with\\%wild%cards_"));
```

Comment:
❓ Should we test this for escaped `?` as well? Plus, what does the RDBMS do with those escaped `*`s and `?`s? Does it take those literally in the end or how is this interpreted?
```suggestion
        Arguments.of("some\\*query\\_with%wild*car\\?ds?", "some\\*query\\_with\\%wild%car\\?ds_"));
```